### PR TITLE
Add firefox/retention/thank-you.lang

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

**Comment by TheoChevalier** [comment]
I wonder, why is the tweet not localized?

**Comment by flodolo** [comment]
Ugh, good point :-\

File: en-US/firefox/retention/thank-you.lang [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+# Action (sharing a link to Firefox) takes only 1 minute
+;1 min share
+1 min share
+
+
+;Send a tweet
+Send a tweet
+
+
+# Tweet: keep it short, since emojis and URL are used as well
```
Comment (TheoChevalier) [inline en-US/firefox/retention/thank-you.lang]: After playing with the page, I think we can use a LENGTH_LIMIT=107

Comment (flodolo) [reply en-US/firefox/retention/thank-you.lang]: Thanks, let's try with an explicit length.

File: en-US/firefox/retention/thank-you.lang [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+# Used as a tweet, keep it under 107 characters (emojis and URL are added to the text).
+;Join me in the fight for an open web by choosing Firefox!
+Join me in the fight for an open web by choosing Firefox!
+
+
+;Make the Internet a safer place
+Make the Internet a safer place
+
+
+# Action (reading the article) takes only 1 minute.
```
Comment (TheoChevalier) [inline en-US/firefox/retention/thank-you.lang]: nit, 4 minutes

**Pull request merged**